src/ccd2rgb.jl

```diff
@@ -0,0 +1,40 @@
+"""
+    ccd2rgb(red::ImageHDU, green::ImageHDU, blue::ImageHDU; stretch::Function = x -> x, shape_out = size(red))
```

Comment:
Do not use the `Function` type for `stretch`: not all "callable" objects in Julia are `Function`s.  Also, use `identity` for the default, as I already suggested you